challenges.yml

```
@@ -0,0 +1,37 @@
+extending_instancedynamicdata:
+  question: |
+    Suppose we have a vue component that looks like this:
+
+    ```javascript
+    const app = new Vue({
+      el: '#app',
+      template: '<span class="title">{{ title }}</span>
+    });
+    ```
+
+    and the `title` is a data property that resides in that same component,
+    how would you go about initializing this data?
+  explanation: |
+    That's right! `data` is a `function` that returns an object. The properties
```

```suggestion
    That's right! `data` is a function that returns an object. The properties
```

function is not really a technical word in this case :)